views/apachesolr_angularjs_search_handler_field_delete_link.inc

```diff
@@ -40,7 +40,7 @@ class apachesolr_angularjs_search_handler_field_delete_link extends views_handle
   function render($values) {
     $link_text = $this->options['link_text'];
     $sid = $values->sid;
-    $form = drupal_get_form('apachesolr_angularjs_search_delete_form', $sid, $link_text);
+     $form = drupal_get_form('apachesolr_angularjs_search_delete_form_'.$sid, $sid, $link_text);
```

Comment:
Could you please remove the extra space at the beginning of the line and add a space to both sides of the concatenation dot?